TESTING.md

```diff
@@ -109,6 +109,35 @@ mvn clean install -pl janusgraph-es -Delasticsearch.docker.image=elasticsearch
 mvn clean install -pl janusgraph-es -Delasticsearch.docker.version=1.5.1 -Delasticsearch.docker.image=elasticsearch
 ```
 
+### Running CQL Tests with a Cassandra
+
+**Note** Running Cql tests requires docker.
+
+```bash
+mvn clean install -pl janusgraph-cql -Pcassandra2
+mvn clean install -pl janusgraph-cql -Pcassandra3
+```
+
+Finally the `cassandra.docker.version` property can be used to test against arbitrary Cassandra versions. This is more complicated however because of differences across major versions in required server settings and Docker image names. The examples below illustrate the differences based on the Cassandra major version.
```

Comment:
(nitpick) _Finally_ only makes sense if it's the last part of a longer section, but this is currently the first sentence in this section.
Also please split long lines.
Apart from that, I don't understand what's complicated here. Why is the Docker image name different for different versions? The only image name I see is _cassandra_.